packages/now-routing-utils/test/index.spec.js

```diff
@@ -67,6 +75,32 @@ describe('normalizeRoutes', () => {
     assert.deepStrictEqual(normalized.routes, routes);
   });
 
+  test('should show proper error for invalid handle: error', () => {
+    const routes = [
+      {
+        handle: 'error',
+        status: 404,
+        invalid: 'value',
+        dest: '/404',
+        src: '^.*$',
+      },
```

Comment:
This structure was never discussed...

It should look like this:
```
{
  handle: 'error',
}
{
  src: '^/.*$',
  status: 404,
  dest: '/404',
  continue: true
},
```

Reply:
Updated